app/script/entity/message/ContentMessage.js

```diff
@@ -109,7 +109,9 @@ z.entity.ContentMessage = class ContentMessage extends z.entity.Message {
   }
 
   isSelfMentioned() {
-    return this.has_asset_text() && this.get_first_asset().isSelfMentioned();
+    return (
+      this.has_asset_text() && this.assets().some(assetEntity => assetEntity.is_text() && assetEntity.isSelfMentioned())
+    );
```

Comment:
Sorry to be a little bit annoying but I am not a fan of single lines if it doesn't fit the single line 🤷‍♂️ 

I would prefer:

````
return this.has_asset_text()
  ? this.assets().some(assetEntity => assetEntity.is_text() && assetEntity.isSelfMentioned())
  : false;
````

or

````
if (this.has_asset_text()) {
  return this.assets().some(assetEntity => assetEntity.is_text() && assetEntity.isSelfMentioned());
}
````

Where the second one would return void instead of false in case there is no text asset.